docs/RFCS/range_leases.md

```diff
@@ -47,43 +49,44 @@ We introduce a new node lease table at the beginning of the keyspace
-meta1 and meta2) use the current, per-range lease mechanism to
-avoid circular dependencies. This table maps node IDs to an epoch
-counter and a lease expiration timestamp.
-
-The range lease is moved from a special raft command (which
-writes to a range-local, non-transactional range lease key) to a
-transactional range-local key (similar to the range descriptor). The
-range lease identifies the node that holds the lease and its
-epoch counter. It has a start timestamp but does not include an
-expiration time. The lease record is always updated in a distributed
-transaction with the node lease record to ensure that the epoch
-counter is consistent and the start time is greater than the prior
-lease holder’s node lease expiration (plus the maximum clock offset).
+meta1 and meta2) use the current, per-range lease mechanism to avoid
+circular dependencies. This table maps node IDs to an epoch counter
+and a lease expiration timestamp.
+
+The range lease is moved from a special raft command (which writes to
+a range-local, non-transactional range lease key) to a transactional
+range-local key (similar to the range descriptor). The range lease
+identifies the node that holds the lease and its epoch counter. It has
+a start timestamp but does not include an expiration time. The range
+lease record is always updated in a distributed transaction with the
+node lease record to ensure that the epoch counter is consistent and
+the start time is greater than the prior range lease holder’s node
+lease expiration (plus the maximum clock offset).
 
 Each node periodically performs a conditional put to its node lease to
 increase the expiration timestamp and ensure that the epoch has not
 changed. If the epoch does change, *all* of the range leases held by
 this node are revoked. A node *must not* propose any commands with a
 timestamp greater than the latest expiration timestamp it has written
-to the lease table.
+to the node lease table.
 
 A range lease is valid for as long as the node’s lease has the same
 epoch. If a node is down (and its node lease has expired), another
-node may revoke its lease(s) by incrementing the epoch. Once this is
-done the old lease is invalidated and a new node may claim the lease.
+node may revoke its lease(s) by incrementing the node lease
+epoch. Once this is done the old range lease is invalidated and a new
+node may claim the range lease.
 
-A node may give up its own lease without incrementing the epoch
-counter by means of a conditional put to set the leaseholder to
-zero. This is necessary in the case of rebalancing when the node that
-holds the lease is being removed. Leases can be transferred using a
-similar mechanism, for example to respond to changing geographic
-traffic patterns.
+A node may give up a range lease it owns without incrementing the
+epoch counter by means of a conditional put to the range lease record
+to set the leaseholder to zero. This is necessary in the case of
```

Comment:
Reworked this paragraph to only talk about lease transfers and `AdminTransferLease`. Giving up a lease before it expires without transferring it is not particularly interesting.